streaming-binlog/src/main/java/org/openmrs/analytics/FhirConverter.java

```diff
@@ -97,8 +92,7 @@ public void process(Exchange exchange) {
 			// TODO: check how this can be signalled to Camel to be retried.
 			return;
 		}
-		String resourceJson = parser.encodeResourceToString(resource);
```

Comment:
Oops; thanks for catching this; not sure we were converting this to `String` just to convert back to `Resource` in `FhirStoreUtil`.